pandas/io/excel/_util.py

```diff
@@ -85,7 +86,7 @@ def _excel2num(x):
     return index - 1
 
 
-def _range2cols(areas):
+def _range2cols(areas: str) -> List[int]:
```

Comment:
are you going to make separate PRs for stuff like this that works immediately?

Reply:
not sure of the approach at this time. not really made much impact on the complete task yet.

a couple of things to consider:

to realize the intended benefit of additional PR checking, we'll need to enable the check-untyped-defs globally and create a blacklist.
 
In order to remove modules from the blacklist quicker, i've started with the modules containing the least errors, but these would be the best choices for community engagement, so happy if that's done by the community and just use this branch as a reference for a potential solution.